gitrevise/tui.py

```diff
@@ -88,6 +88,14 @@ def build_parser() -> ArgumentParser:
         action="store_true",
         help="interactively cut a commit into two smaller commits",
     )
+
+    mode_group = parser.add_mutually_exclusive_group()
+    mode_group.add_argument(
+        "--gpg-sign", "-S", action="store_true", default=None, help="GPG sign commits",
```

Comment:
That's great, thanks!  I am going to include this, joyfully.

Reply:
I think I found a way to also make `git revise -eS HEAD` work:

```python
            if (arg.startswith("-") and arg.endswith("S")) or arg == "--gpg-sign":
                argv.insert(i+1, '')
```

OTOH I realized that this hack makes the usage from `git revise -h` inconsistent.
So maybe it's better to stick with standard argparse even though that's kind of ugly. I'm not sure.

Reply:
I found a way to make `git revise -eS HEAD` work as well:

```
            if (arg.startswith("-") and arg.endswith("S")) or arg == "--gpg-sign":
                argv.insert(i+1, '')
```

However, because of this hack the usage information from `git revise -h` is misleading:
```
  --gpg-sign GPG_SIGN, -S GPG_SIGN
```
Currently, GPG_SIGN would be interpreted as revision argument.

Maybe it's better to stick to standard argparse, even though it feels super weird compared to the GNU style options that Git uses. Or find out how to fix the usage string, I'm not sure.